.hlint.yaml

```
@@ -1,4 +1,18 @@
 - arguments: [-XTypeApplications, -XNumericUnderscores]
 # Often, code is more readable with extra parens, or less readable with list comprehensions
 - ignore: {name: "Redundant bracket"}
-- ignore: {name: "Use list comprehension"}
\ No newline at end of file
+- ignore: {name: "Use list comprehension"}
+
+# Import Preferences
+- modules:
+  - {name: Data.Set, as: Set, message: "Use complete name for qualified import of Set"}
+  - {name: Data.Map, as: Map, message: "Use complete name for qualified import of Map"}
+  - {name: Data.Text, as: Text, message: "Use complete name for qualified import of Text"}
+
+# Styling Preferences
+- suggest: {lhs: "asum [x, y]", rhs: x <|> y}
+- suggest: {lhs: return x, rhs: pure x}
```

I think this is causing a rewrite from `return $ f a` to `pure (f a)`.  I think rule can be rewritten to fix that.
```suggestion
- suggest: {lhs: return, rhs: pure}
```